src/applications/claims-status/components/claim-overview-tab/ClaimPhaseStepper.jsx

```diff
@@ -146,13 +141,7 @@ export default function ClaimPhaseStepper({
             You’ll be able to view and download your decision letter on the
             status page for this claim.
           </p>
-          <Link
-            aria-label="Go to the claim letters page"
-            title="Go to the claim letters page"
-            to="/your-claim-letters"
-          >
-            Go to the claim letters page
-          </Link>
+          <Link to="/your-claim-letters">Go to the claim letters page</Link>
```

Comment:
Updated active link to no longer have a title or aria-label since its not needed